sdk/attestation/Azure.Security.Attestation/tests/AttestationClientTestEnvironment.cs

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.Identity;
+
+
+// licy":"CnZlcnNpb249MS4 => version=1.
+// licy":"DQp2ZXJzaW9uPTE => version=1
```

Comment:
?

Reply:
Oops - debugging goo accidentally left in :(.